src/client/async.rs

```diff
@@ -481,15 +486,18 @@ impl ClientService {
     /// the connection for transmission to the server (i.e. `start_request`).
     /// Also returns the sender end of the channel to which the response is to be transmitted,
     /// once received.
-    fn create_request(&self, async_req: AsyncRequest)
-            -> (RequestStream<'static, 'static, DefaultStream>, Sender<StaticResponse>) {
+    fn create_request(&self,
+                      async_req: AsyncRequest)
+                      -> (RequestStream<'static, 'static, DefaultStream>,
+                          Sender<StaticResponse>) {
         let mut headers: Vec<Header> = Vec::new();
         headers.extend(vec![
             Header::new(b":method", async_req.method),
             Header::new(b":path", async_req.path),
             Header::new(b":authority", self.host.clone()),
             Header::new(b":scheme", self.conn.scheme().as_bytes().to_vec()),
-        ].into_iter());
+        ]
+                           .into_iter());
```

Comment:
This looks like a `rustfmt` bug. There's no way this should be expected formatting? :)